webapp/src/Entity/Submission.php

```diff
@@ -38,7 +38,8 @@
 #[UniqueEntity(fields: 'externalid')]
 class Submission extends BaseApiEntity implements
     HasExternalIdInterface,
-    ExternalIdFromInternalIdInterface
+    ExternalIdFromInternalIdInterface,
+    PrefixedExternalIdInShadowModeInterface
```

Comment:
As stated above, this case can happen if people resubmit after changing the code, useful for both jury as analysts.